Styles/Classes/LayerStyle.swift

```diff
@@ -2,23 +2,68 @@
 
 import UIKit
 
+extension UIRectCorner {
+    @available(iOS 11.0, *)
+    var maskedCorners: CACornerMask {
+        var mask: CACornerMask = []
+        if self.contains(.topLeft) {
+            mask.formUnion(CACornerMask.layerMinXMinYCorner)
+        }
+        if self.contains(.topRight) {
+            mask.formUnion(CACornerMask.layerMaxXMinYCorner)
+        }
+        if self.contains(.bottomLeft) {
+            mask.formUnion(CACornerMask.layerMinXMaxYCorner)
+        }
+        if self.contains(.bottomRight) {
+            mask.formUnion(CACornerMask.layerMaxXMaxYCorner)
+        }
+        if self.contains(.allCorners) {
+            mask = [
+                .layerMinXMinYCorner,
+                .layerMaxXMinYCorner,
+                .layerMinXMaxYCorner,
+                .layerMaxXMaxYCorner
+            ]
+        }
+        return mask
+    }
+}
+
 public final class LayerStyle: NSObject {
     public enum Property {
         case borderColor(UIColor)
         case borderWidth(CGFloat)
-        case cornerRadius(CGFloat)
```

Comment:
i would keep both options whereas `cornerRadius` renamed to `roundCorners(CGFloat)`